cognee/tasks/repo_processor/local_script_dependencies.py

```diff
@@ -0,0 +1,83 @@
+from typing import List, Dict, Optional
+import jedi
+import parso
+import sys
+from pathlib import Path
+from parso.tree import BaseNode
+
+
+def get_code_entities(node: parso.tree.NodeOrLeaf) -> List[Dict[str, any]]:
```

Comment:
Doesn't this function need to be async?

Reply:
Turns out it does, thanks for the question! Updated it.